README.md

```diff
@@ -47,6 +47,17 @@ In this example, the Makefile would look for an implementation in ocaml_exercise
 
 All pull requests are run through a Travis CI build, which compiles and runs tests.
 
+## Canonical test data
+
+If you are implementing an existing test, there may be "canonical data" for it in the [x-common](https://github.com/exercism/x-common) repository.
+An example: https://github.com/exercism/x-common/blob/master/exercises/bracket-push/canonical-data.json
+
+You should base your tests off this data, in order to provide consistency across different language tracks.
+
+There is a test generator which can read these json file, together with a template file that you would need to create,
+and creates test ocaml files. It is a work in progress, so may not work for every exercise. If you wish to use it, there
```

Comment:
I am not a native english speaker, but should this read: '... and create~~s~~ test ocaml files' without the 's'?